docs/AQL/master03-syntax.adoc

```diff
@@ -353,6 +354,91 @@ WHERE
    NOT EXISTS c/content[openEHR-EHR-ADMIN_ENTRY.discharge.v1]
 --------
 
+== Functions
+
+=== Aggregate functions
+
+Aggregate functions calculates a single result from a set of _input values_, aggregated from an _expression_ over the result set.
+These expressions are using <<_aql_identified_paths, identified paths>>
+so that the functions operates on a set of <<_built_in_types,built in (primitive)>> type values.
+Some of these functions uses a `DISTINCT` keyword to consider only distinct values from the set.
+Unless specified, this functions omits the `NULL` values.
```

Comment:
```suggestion
Aggregate functions calculate a single result from a set of _input values_, aggregated from an _expression_ over the result set.
These expressions are using <<_aql_identified_paths, identified paths>>
so that the functions operate on a set of <<_built_in_types,built in (primitive)>> type values.
Some of these functions support the `DISTINCT` keyword to consider only distinct values from the set.
Unless specified otherwise, these functions ignore `NULL` values.
```